### PR TITLE
feat: adds usePlausible custom hook

### PR DESCRIPTION
**Pull request by itofarina**

### CODE REVIEW

**Comment by jurajpiar** [comment]
@farinalvaro what does go to the data domain? https://github.com/rsksmart/rif-marketplace-ui/pull/789/files#diff-d3c6c3079396f5d28b31f453c183751328b36c1513068e50cb293eeb9f9d4eedR55

File: src/components/organisms/tracking/PrivacySettingsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+const PrivacySettingsModal: FC = () => {
+  const classes = useStyles()
+  const settingsString = localStorage.getItem('privacySettings')
+  const [isOpen, setIsOpen] = useState(true)
+  const [settings, setSettings] = useState<PrivacySettings>()
+
+  usePlausible({
+    activated: Boolean(settings?.tracking),
+    dataDomain: 'WHAT_GOES_HERE?', // FIXME: add data domain
```
Comment (farinalvaro) [inline src/components/organisms/tracking/PrivacySettingsModal.tsx:55]: Sorry, I forgot to mention. 
`marketplace.rifos.org` should go here as we are going to start tracking only mainnet activity.

File: src/components/organisms/tracking/PrivacySettingsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+              Privacy settings
+            </Typography>
+          </Box>
+        </GridItem>
+        <GridItem xs={12}>
+          <Box
+            style={{
+              padding: 10,
+              paddingRight: 75,
+            }}
```
Comment (farinalvaro) [inline src/components/organisms/tracking/PrivacySettingsModal.tsx:98]: Same about moving styles to the `useStyles` hook, please

**Review by itofarina — COMMENTED** [review]
Apart from those little (and may be not very relevant :) ) comments it LGTM. Can't approve because I opened it.
Nice job!

File: src/components/organisms/tracking/PrivacySettingsModal.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
+import React, { FC, useEffect, useState } from 'react'
+import {
+  Box,
+  createStyles, Link, makeStyles, Portal, Theme, Typography,
+} from '@material-ui/core'
```
[the remark this answers is not in the document]
Comment (jurajpiar) [reply src/components/organisms/tracking/PrivacySettingsModal.tsx:5]: Good point, thanks.

File: src/components/organisms/tracking/PrivacySettingsModal.tsx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    boxShadow: theme.shadows[5],
+    color: colors.gray5,
+  },
+  titleBox: {
+    marginTop: 0,
+    height: 46,
+    borderRadius: '20px 20px 0px 0px',
+    backgroundColor: colors.gray3,
+    color: '#4D4C4C',
+    justifySelf: 'center',
```
[the remark this answers is not in the document]
Comment (jurajpiar) [reply src/components/organisms/tracking/PrivacySettingsModal.tsx:35]: we can move that into a low-priority issue as this needs to be shipped and you're not in today to release a new version of rif-ui.

**Review by 0xartem — APPROVED** [review]
LGTM, let's move to staging

**Pull request merged**